### ISSUE TITLE
please help me to start 

### ISSUE THREAD

**Issue by vahidalvandi** [nazmussakib44/laravel-react-starter-kit#2]
hello , thank you for publish this project 

Is it possible to put more complete codes on the frontend side, especially when logging in and storing the localstorage token and controlling authentication in the dashboard?

**Comment by nazmussakib44** [comment]
This project is pretty much complete. Though the UI pages are not functional yet. You may let me know exactly which feature you need.

**Comment by vahidalvandi** [comment]
I tested it not store local storage token  and not logined for dashbord  route

**Comment by nazmussakib44** [comment]
Yes, the request_login redux action is there to work further. UI pages are not implemented. I am thinking of adding Login, Registration, Dashboard & dynamic Nav menu with tailwind but can't promise a date.

**Comment by vahidalvandi** [comment]
Only please show me login ,logout , dashboard  in this project
